### PR TITLE
Wording clarification

### PR DESCRIPTION
**Pull request by paigehargrave**

### CODE REVIEW

**Review by jeff-1amstudios — APPROVED** [review]

**Comment by paigehargrave** [comment]
@jeff-1amstudios - Thank you!

File: modules/ROOT/pages/install-sys-reqs.adoc [hunk 1]
```diff
@@ -26,12 +26,11 @@ a different operating system version or distribution, the Runtime Fabric install
...
 Only x86 and x64 architectures are supported.
 
-Runtime Fabric reserves up to 0.5 cores per worker node for internal services. This is important 
+Runtime Fabric can reserve up to 0.5 cores per worker node for internal services. This is important 
 when determining the amount of CPU cores and worker nodes needed.
 
-The Runtime Manager UI displays the amount of available CPU across worker nodes. For example,
-given a cluster with 3 worker nodes, each with 2 cores, Runtime Fabric reserves a total of 0.9
-cores (0.3 x 3). The number of available cores displayed in the Runtime Manager is 5.1 instead
+The Runtime Manager UI displays the amount of available CPU across worker nodes. For example, you might have a cluster with 3 worker nodes and 2 cores for each worker node. If Runtime Fabric reserves a total of 0.9
```
Comment (sundippatel) [inline modules/ROOT/pages/install-sys-reqs.adoc]: I think we can be more clear here and lead with suggesting Runtime Fabric is reserving 0.3 cores per worker node.

"For example, you may have a Runtime Fabric with 3 worker nodes and 2 CPU cores for each worker node. If Runtime Fabric reserves 0.3 cores per worker node for a total of 0.9 cores in total, the number of available cores displayed in Runtime Manager would be 5.1 vCPU instead of 6 vCPU."

File: modules/ROOT/pages/install-sys-reqs.adoc [hunk 2]
```diff
@@ -26,12 +26,10 @@ a different operating system version or distribution, the Runtime Fabric install
 == Hardware Requirements
 Only x86 and x64 architectures are supported.
 
-Runtime Fabric can reserve up to 0.5 cores per worker node for internal services. This is important 
+Runtime Fabric reserves 0.3 cores per worker node for internal services. This is important
```
Comment (sundippatel) [inline modules/ROOT/pages/install-sys-reqs.adoc]: Should revert this change, and I think we're good.

**Review by dhirsch1 — APPROVED** [review]

File: modules/ROOT/pages/install-sys-reqs.adoc (hunk 2 above)
Comment (paigehargrave) [reply modules/ROOT/pages/install-sys-reqs.adoc]: Got it - thank you!
